Data-Structures/Vectors/Vector2.js

```diff
@@ -4,75 +4,6 @@
  * https://en.wikipedia.org/wiki/Vector_(mathematics_and_physics)).
  */
 
-/*
-Doctests
-
-// Test equalsExactly-method
-> new Vector2(1, 0).equalsExactly(new Vector2(1, 0))
-true
-> new Vector2(1.23, 4.56).equalsExactly(new Vector2(0, 0))
-false
-
-// Test equalsApproximately-method
-> new Vector2(1, 0).equalsApproximately(new Vector2(1, 0.0000001), 0.000001)
-true
-> new Vector2(1.23, 4.56).equalsApproximately(new Vector2(1.24, 4.56), 0.000001)
-false
-
-// Test add-method
-> new Vector2(1, 0).add(new Vector2(0, 1)).equalsApproximately(new Vector2(1, 1), 0.000001)
-true
-> new Vector2(-3.3, -9).add(new Vector2(-2.2, 3)).equalsApproximately(new Vector2(-5.5, -6), 0.000001)
-true
-
-// Test subtract-method
-> new Vector2(1, 0).subtract(new Vector2(0, 1)).equalsApproximately(new Vector2(1, -1), 0.000001)
-true
-> new Vector2(234.5, 1.7).subtract(new Vector2(3.3, 2.7)).equalsApproximately(new Vector2(231.2, -1), 0.000001)
-true
-
-// Test multiply-method
-> new Vector2(1, 0).multiply(5).equalsApproximately(new Vector2(5, 0), 0.000001)
-true
-> new Vector2(3.41, -7.12).multiply(-3.1).equalsApproximately(new Vector2(-10.571, 22.072), 0.000001)
-true
-
-// Test length-method
-> new Vector2(1, 0).length()
-1
-> new Vector2(-1, 1).length()
-Math.sqrt(2)
-
-// Test normalize-method
-> new Vector2(1, 0).normalize().equalsApproximately(new Vector2(1, 0), 0.000001)
-true
-> new Vector2(1, -1).normalize().equalsApproximately(new Vector2(Math.sqrt(2) / 2, -Math.sqrt(2) / 2), 0.000001)
-true
-
-// Test distance-method
-> new Vector2(0, 0).distance(new Vector2(0, -1))
-1
-> new Vector2(1, 0).distance(new Vector2(0, 1))
-Math.sqrt(2)
-
-// Test dotProduct-method
-> new Vector2(1, 0).dotProduct(new Vector2(0, 1))
-0
-> new Vector2(1, 2).dotProduct(new Vector2(3, 4))
-1 * 3 + 2 * 4
-
-// Test rotate-method
-> new Vector2(0, -1).rotate(Math.PI / 2).equalsApproximately(new Vector2(1, 0), 0.000001)
-true
-> new Vector2(1.23, -4.56).rotate(Math.PI).equalsApproximately(new Vector2(-1.23, 4.56), 0.000001)
-true
-
-// Test angleBetween-method
-> new Vector2(1, 0).angleBetween(new Vector2(0, 1))
-Math.PI / 2
-> new Vector2(1, 0).angleBetween(new Vector2(1, -1))
--Math.PI / 4
-*/
 class Vector2 { // eslint-disable-line no-unused-vars
```

Comment:
The `eslint` comment is not needed anymore since we actually export the class.

Furthermore: You can simply `export class Vector2` instead of explicitly exporting the symbol down below - but that's basically just a matter of preference.